src/App.test.jsx

```diff
@@ -0,0 +1,45 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+
+import App from './App';
+
+describe('App', () => {
+  it('Should render the character detail page of Summer Smith then allow user to navigate to list of characters', async () => {
+    render(
+      <MemoryRouter initialEntries={['/characters/3']} initialIndex={2}>
+        <App />
```

Comment:
The initial index correlates to the initialEntries array, which in this case would be zero :)